app/controllers/concerns/umrdr/works_controller_behavior.rb

```diff
@@ -4,6 +4,26 @@ module WorksControllerBehavior
     include Sufia::Controller
     include Sufia::WorksControllerBehavior
     include CurationConcerns::CurationConcernController
+       
+    # override curation concerns, add form fields values
+    def build_form
+    
+      super
+      st = EDTF.parse(@form["date_coverage_from"])
+      if (!st.eql? nil)
```

Comment:
Nil checks like this are inadvisable.  This can just be `if st`.  
https://github.com/bbatsov/ruby-style-guide#no-non-nil-checks

Also, consider a more descriptive variable name.